test/contracts/front/roles.js

```diff
@@ -249,6 +249,13 @@ describe('Roles Contract', () => {
         expect(await getRole(apollo)).to.equal(APOLLO.toString());
       });
 
+      it('allows to onboard with bigger stake', async () => {
```

Comment:
Perhaps move this test after `adds node to validator set and block rewards`. It kind of builds upon it and `stores the role and deposit` - it contains assertions for both checking if onboarding is successful and for beneficiaryShare.